resources/bundles/org.eclipse.core.filesystem/src/org/eclipse/core/filesystem/IFileStore.java

```diff
@@ -449,6 +477,16 @@ public interface IFileStore extends IAdaptable {
 	 */
 	public OutputStream openOutputStream(int options, IProgressMonitor monitor) throws CoreException;
 
+	/** @since 1.11 **/
```

Comment:
Javadoc is missing but should be added for new API.

Reply:
ok, added